monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentControllerV1.java

```diff
@@ -210,18 +213,41 @@ public OutputColumns getOutputColumns(@PathVariable(PARAMETER_NAME_TENANT) final
     @RequestMapping(value = "/{name}", method = RequestMethod.PUT, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     public ProcessStatelessOutput processStateless(@PathVariable(PARAMETER_NAME_TENANT) final String tenant,
             @PathVariable(PARAMETER_NAME_NAME) final String name,
+            @RequestParam(value = PARAMETER_NAME_OUTPUT_STYLE, required = false, defaultValue = "tabular") OutputStyle outputStyle,
```

Comment:
At this level I would not use the enum because that introduces an unfortunate link between java enum naming conventions (uppercase and underscore etc) which does not map well to http paramo. Rather I would make the string-to-enum conversion immediately after in your controller, and then pass it on to a method which takes the enum. That way you separate http interface (unsafe, dynamic) with java interface (safe, typed).